deployment/terraform/resources/batch.tf

```diff
@@ -5,6 +5,9 @@ resource "azurerm_storage_account" "pctasks-batch" {
   account_tier                    = "Standard"
   account_replication_type        = "LRS"
   allow_nested_items_to_be_public = false
+  
+  # Shared access key gets disabled via deploy script
+  # shared_access_key_enabled       = false
```

Comment:
Should this be uncommented?

Reply:
No, but it can be deleted. The deploy script will set that on/off and not have it managed by tf.